lte/gateway/python/magma/monitord/icmp_monitoring.py

```diff
@@ -79,13 +76,9 @@ def _save_ping_response(self, target_id: str, ip_addr: str,
             self._module.save_ping_response(target_id, ip_addr, ping_resp)
 
     async def _run(self) -> None:
```

Comment:
For my understanding: does this still run in a loop? It looks like you removed the explicit loop